scripts/ubuntu-16.04_tkgplus-1.17_weave-2.5.2_rev3/cust.sh

```diff
@@ -117,6 +127,22 @@ truncate -s 0 /etc/machine-id
 rm /var/lib/dbus/machine-id || :
 ln -fs /etc/machine-id /var/lib/dbus/machine-id || : # dbus/machine-id is symlink pointing to /etc/machine-id
 
+# create /root/kubeadm-defaults.conf
+echo "---
+apiVersion: kubeadm.k8s.io/v1beta2
+kind: ClusterConfiguration
+dns:
+  type: CoreDNS
+  imageRepository: k8s.gcr.io
```

Comment:
If we keep `vmware.io` as mentioned above, L136, 140, 142 will need to change to `vmware.io`